### DISCUSSION TITLE
Way to disable the DSO catalog 3.12 warning

### DISCUSSION THREAD

**Discussion by deLabe** [Stellarium/stellarium#1714] [Q&A] [answered]
Can a way to disable the DSO catalog 3.12 warning be added?

**Comment by alex-w** [comment]
Please more details: which warning exactly

**Reply by deLabe** [reply]
![DSO Warning](https://user-images.githubusercontent.com/85290505/120643416-23a88900-c444-11eb-9c97-c697a7e75476.png)

**Comment by gzotti** [comment]
The warning is here to guide you towards updating. 

"Is there a way to disable that stupid emergency brake?"

**Reply by deLabe** [reply]
My first thought was to update. When I went to do that at [https://github.com/Stellarium/stellarium-data/releases](url), I found this: WARNING: Stellarium MAY run slow with the extended edition of DSO catalog! I do not need a million objects. I do wish my night sky was dark enough to make that useful. 

Also Stellarium does already display other warning messages that display briefly and maybe only one time. (I've noticed this when switching between releases.)  I suggest that would be a better strategy than this pop-up window which prevents the program from loading until you acknowledge it.

**Reply by gzotti** [reply, marked as answer]
At some earlier point in time you have installed version 3.4. Instructions for "placing a new catalog" in the right place can be found in the User Guide, section 5.5.2. We have not written explicit instructions for "not placing" the catalog there, but I hope you can work out what to do with the file currently found there.   

One-time messages exist. They instruct what to do, and may vanish after acknowledment of consequences (esp. bad graphics). The wrong DSO catalog will cause various errors, so this warning will not go away.
